From 5f7f1f1e3ca21613484fa2c9ea47f3661dc4399e Mon Sep 17 00:00:00 2001
From: "djm@openbsd.org" <djm@openbsd.org>
Date: Wed, 24 Jun 2026 06:55:12 +0000
Subject: [PATCH 04/12] upstream: mention a caveat regarding
 GSSAPIStrictAcceptorCheck in

some environments

OpenBSD-Commit-ID: aa7158d8f22cb34063c1c2d3cbcf30a9489847c2
(cherry picked from commit 8058c5bdb507591b79ec926221fbe6fcc296d432)
CVE-2026-59998
---
 sshd_config.5 | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sshd_config.5 b/sshd_config.5
index 3f5e29812d3f..22b9039d25b7 100644
--- a/sshd_config.5
+++ b/sshd_config.5
@@ -769,6 +769,7 @@ machine's default store.
 This facility is provided to assist with operation on multi homed machines.
 The default is
 .Cm yes .
+This option may not be effective in Windows Active Directory environments.
 .It Cm HostbasedAcceptedAlgorithms
 Specifies the signature algorithms that will be accepted for hostbased
 authentication as a list of comma-separated patterns.
-- 
2.55.0.dirty

